vmm/src/lib.rs

```diff
@@ -390,6 +390,25 @@ struct VmMigrationConfig {
     memory_manager_data: MemoryManagerSnapshotData,
 }
 
+#[derive(Debug, Clone)]
+pub struct VmmVersionInfo {
+    pub build_version: String,
+    pub version: String,
+}
+
+impl VmmVersionInfo {
+    pub fn new(build_version: &str, version: &str) -> Self {
+        Self {
+            build_version: build_version.to_owned(),
+            version: version.to_owned(),
+        }
+    }
+
+    pub fn dummy() -> Self {
+        Self::new("dummy", "dummy")
+    }
```

Comment:
Considering it will only be used for unit test code, why don't you simply call `new()` and get rid of the `dummy` method?

Reply:
Makes perfect sense, thanks for pointing it out.